companies/volkswagen-ag.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "volkswagen-ag",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Volkswagen AG",
+    "address": "Volkswagen AG\nPrivacy Team\nBerliner Ring 2\n38440 Wolfsburg\nGermany",
+    "email": "privacy@volkswagen.de",
+    "web": "https://www.volkswagenag.com/",
+    "sources": [
+        "https://datenschutz.volkswagen.de/"
+    ],
+    "suggested-transport-medium": "email",
+    "comments": [
```

Comment:
This information is already mentioned in the record, so no need for the comment.